src/Transports.Subscriptions.WebSockets/Transports.Subscriptions.WebSockets.csproj

```diff
@@ -2,10 +2,6 @@
   
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;netstandard2.0</TargetFrameworks>
-    <AssemblyName>GraphQL.Server.Transports.WebSockets</AssemblyName>
```

Comment:
**Attention!**
@pekkah @joemcbride I noticed a naming inconsistency.

Before:
```
ProjectName = Transports.Subscriptions.WebSockets
AssemblyName = PackageName = GraphQL.Server.Transports.WebSockets
```

After:
```
ProjectName = Transports.Subscriptions.WebSockets
AssemblyName = PackageName = GraphQL.Server.Transports.Subscriptions.WebSockets
```

All other projects consistently add `GraphQL.Server` to the name.
Moreover 
```
<Description>WebSockets transport for subscriptions</Description>
```
clearly says that this is a transport for subscriptions, not some general transport on web sockets. So I propose to change package name to `GraphQL.Server.Transports.Subscriptions.WebSockets`. See https://ci.appveyor.com/project/graphql-dotnet-ci/server/builds/31132815/artifacts